guides/dev/package_managers/README.md

```diff
@@ -28,6 +28,41 @@ It is thus recommended to use of the following strategies:
 
 Both options are explained in more detail in the [Setting CKEditor Configuration](http://docs.ckeditor.com/#!/guide/dev_configuration) article.
 
+## npm
+
+This article assumes that you have **npm** already installed and added to your `PATH`. If this is not the case, please follow the [installation guide](https://docs.npmjs.com/getting-started/installing-node).
+
+### Usage
+
+In order to fetch the most recent build execute the following command:
+
+	npm install ckeditor
+
+By default CKEditor will be placed in the `node_modules/ckeditor` directory.
+
+### Adding CKEditor as Dependency Using the `package.json` File
```

Comment:
Hmm to be honest I'm not sure if we need that part, this is kinda duplicating npm docs. Maybe we'd be better just by linking in previous section to npm docs? Or just skip it at all.
RFC @AnnaTomanek @wwalc @Reinmar @fredck